src/components/shared/SectionStepInfo/SectionStepInfo.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import style from './SectionStepInfo.module.scss';
+
+function SectionStepInfo(props) {
+  const { section, description } = props;
+  return (
+    <div className={`${style.sectionWrapper}`}>
+      <p className={`wmnds-m-b-xs ${style.section}`}>{section}</p>
+      <p className={`wmnds-m-b-sm ${style.description}`}>
+        <strong>{description}</strong>
+      </p>
+    </div>
+  );
```

Comment:
Not sure these classes are needed, see progress indicator: https://designsystem.wmnetwork.co.uk/components/form-elements/